### PR TITLE
Add changelog and bump versions

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: servant-client/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 [The latest version of this document is on GitHub.](https://github.com/haskell-servant/servant/blob/master/servant-client/CHANGELOG.md)
 [Changelog for `servant` package contains significant entries for all core packages.](https://github.com/haskell-servant/servant/blob/master/servant/CHANGELOG.md)
 
+0.13
+----
+
+0.13
+----
```
Comment (alpmestan) [inline servant-client/CHANGELOG.md]: typo here? 2 times `0.13`.

File: servant-foreign/CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,9 @@
 
 ### Breaking changes
 
-* Make foreign client Header arguments have the representation of 'Maybe' in those languages
+- *servant* Add `Servant.API.Modifiers`
+  ([#873](https://github.com/haskell-servant/servant/pull/873)) which superseded
+- Make foreign client Header arguments have the representation of 'Maybe' in those languages
```
Comment (alpmestan) [inline servant-foreign/CHANGELOG.md]: I think you can drop the whole `which superseeded [...]` bit.

File: servant/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,73 @@
 [The latest version of this document is on GitHub.](https://github.com/haskell-servant/servant/blob/master/servant/CHANGELOG.md)
 
+0.13
+----
+
+### Significant changes
+
+- Streaming endpoint support.
+  ([#836](https://github.com/haskell-servant/servant/pull/836))
```
Comment (alpmestan) [inline servant/CHANGELOG.md]: Might be nice to link to the relevant section in the tutorial?

File: servant/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,73 @@
...
+- *servant* Mono-kind-ise modifiers, resulting in better error messages.
+  ([#887](https://github.com/haskell-servant/servant/issues/887)
+   [#890](https://github.com/haskell-servant/servant/pull/890))
+
+- *servant* Add `TypeError ... => HasServer`s instances in GHC-8.2 for
+  not saturated modifiers (`Capture "foo" :> ...`) or `->` in place of `:>`.
+  ([#893](https://github.com/haskell-servant/servant/pull/893))
+
+- *Cookbook* example projects at
+  http://haskell-servant.readthedocs.io/en/stable/examples.html
```
Comment (alpmestan) [inline servant/CHANGELOG.md]: that's not where the cookbook lives

File: servant/CHANGELOG.md (hunk 3 above)
Comment (phadej) [reply servant/CHANGELOG.md]: How to make `readthedocs` build branches?

Comment (phadej) [reply servant/CHANGELOG.md]: found.

**Comment by phadej** [comment]
@alpmestan please re-review.

**Pull request merged**